package.json

```diff
@@ -36,7 +35,10 @@
         "start": "react-scripts start",
         "build": "react-scripts build",
         "test": "react-scripts test",
-        "serve": "serve -s build -l 3000"
+        "build:functions": "yarn global add tslint@5.12.0 && yarn global add typescript@3.4 && cd functions && npm install",
```

Comment:
why are we installing global dependencies? cant we just include them as part of the project's dependencies?

Reply:
I tried very very hard to not need this but for some reason I could not get it to work. The dependencies would be installed locally but the commands could not be found on the travis vm when it ran.